docs/bucket/notifications/README.md

```diff
@@ -231,54 +272,54 @@ client.connect("localhost",1883,60)
 client.loop_forever()
 ```
 
-Execute this example python program to watch for MQTT events on the console.
+Use the following command to execute this program and watch for MQTT events on the console:
 
 ```py
 python mqtt.py
 ```
 
-Open another terminal and upload a JPEG image into ``images`` bucket.
+Open another console and use the following command to upload a JPEG image into ``images`` bucket:
 
-```
+```sh
 mc cp myphoto.jpg myminio/images
 ```
 
-You should receive the following event notification via MQTT once the upload completes.
+Once the upload completes, return to the first console. A response similar to this one should be displayed showing an event notification from MQTT:
 
-```py
-python mqtt.py
-{“Records”:[{“eventVersion”:”2.0",”eventSource”:”aws:s3",”awsRegion”:”",”eventTime”:”2016–09–08T22:34:38.226Z”,”eventName”:”s3:ObjectCreated:Put”,”userIdentity”:{“principalId”:”minio”},”requestParameters”:{“sourceIPAddress”:”10.1.10.150:44576"},”responseElements”:{},”s3":{“s3SchemaVersion”:”1.0",”configurationId”:”Config”,”bucket”:{“name”:”images”,”ownerIdentity”:{“principalId”:”minio”},”arn”:”arn:aws:s3:::images”},”object”:{“key”:”myphoto.jpg”,”size”:200436,”sequencer”:”147279EAF9F40933"}}}],”level”:”info”,”msg”:””,”time”:”2016–09–08T15:34:38–07:00"}
```

Comment:
I think this needs to be closed with a triple tick, it is causing weird rendering issues in the json output below and a few other characters on the page

![minio_readme_md_at_6cfa49f9ce099d5b8f818607b4804bf281064e94_ _minio_minio](https://user-images.githubusercontent.com/41090896/49108579-64a10080-f23d-11e8-990c-7e73788a2dcb.png)